### DISCUSSION TITLE
More contents to this mod

### DISCUSSION THREAD

**Discussion by LordDennis18** [staticalliam7/MindustryPlusPlus#5] [Ideas]
I have an idea to add more additional contents to the mod like new units, turrets, resources and more! Maybe add a new planet so that it's not focused on the campaign.

**Comment by vghfr** [comment]
ok! this isn't actually my main project right now, but feel free to make suggestions

**Comment by LordDennis18** [comment]
-add some new floors like new type of grass or rocks maybe add some plants and liquid resources too, if there will be a campaign.
-it will be cool if you added a few kinds of items that fuels a specific building or can be use as a ammo from the missile array or a turret.
-maybe add 2 or 3 types of unit like ground, air , and naval units.
*Ground- shoots faster and causes more damage but less hp and armor or a slow and heavy machine that deals a lot of damage
*Air- it can be a bomber or full on ground and air support or can call a orbital strike to the nearest enemy
*Naval- that shoots very far but fires very slow and maybe spawns tier 2 or 3 units.
-add bosses for the planets like the "Martian Conqueror" (example only) from Mars
-Turrets
*Groundbreaker
Fires a large beam that can burn and deals a lot of damage.
*Galactum
Fires a small beams but has a very fast fire rate.
*Voldium
Good for air units. Shoots homing laser and explodes on impact.
*Cosmonus
Shoots a powerful orb that can deal huge damage on buildings and group enemies. 

*About the satellites maybe it can be use for calling a orbital strikes or a barrage of missiles and can be control by typing in your coordinates, can be also use for calling allied units from your mod.

**Comment by vghfr** [comment]
Thanks for the ideas! I am restarting the project so I will definitely try to implement these!
